tests/web_api/test_entities.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals, print_function, division, absolute_import
+from http.client import OK
+from nose.tools import assert_equal
+import json
+from . import subject
+
+
+entities_response = subject.get('/entities')
+
+# /entities
+
+
+def test_return_code():
+    assert_equal(entities_response.status_code, OK)
+
+
+def test_response_data():
+    entities = json.loads(entities_response.data.decode('utf-8'))
+    test_description = '''
```

Comment:
Copy-pasting the description makes the test really fragile. A minor edition of the doc would break it.
I'd recommend importing `Household` from the country template and getting the doc from `Household.doc`